all/build.gradle

```diff
@@ -3,21 +3,23 @@ description = "gRPC: All"
 // Make sure that no transitive dependencies are included.
 configurations.compile.transitive = false
 
-for (subproject in rootProject.subprojects) {
-    if (subproject == project) {
-        continue
-    }
+def subprojects = [
+  project(':grpc-auth'),
+  project(':grpc-core'),
+  project(':grpc-netty'),
+  project(':grpc-okhttp'),
+  project(':grpc-protobuf'),
+  project(':grpc-protobuf-nano'),
+  project(':grpc-stub'),
+]
+
+for (subproject in subprojects) {
     evaluationDependsOn(subproject.path)
 }
+evaluationDependsOn(project(':grpc-interop-testing').path)
```

Comment:
I considered something like that, but looping over `jacocoSourceProjects` to call `evaluationDependsOn` is misleading.